src/hotspot/share/runtime/stackOverflow.cpp

```diff
@@ -39,6 +39,11 @@ void StackOverflow::initialize_stack_zone_sizes() {
   // Stack zone sizes must be page aligned.
   size_t page_size = os::vm_page_size();
 
+  // Note: Zone sizes are given via Stack(Red|Yellow|Reserved|Shadow)Pages
+  //  parameters; these names are misleading since they actually contain
+  //  zone size in units-of-4K-size, regardless the actual platform page size,
```

Comment:
s/regardless the/regardless of the/

But this is already covered by the following comment:

  // We need to adapt the configured number of stack protection pages given
  // in 4K pages to the actual os page size. We must do this before setting

Reply:
Okay, I'll remove the comment.